File: Web/ShreyashSriv_ShreyashSrivastava_2125csme1051_2/GettingStarted/IsANumberPrime.cpp
#include<bits/stdc++.h>
using namespace std;

int main(){
    int t;
    cin>>t;
    while(t--){
        int n,count=0;
        cin>>n;
        for(int i=1;i<=n/2;i++){
            if(n%i==0) count++;
            else continue;
        }
        if(count>1) cout<<"not prime"<<endl;
        if(count==1) cout<<"prime"<<endl;
    }
    return 0;
}

File: Web/Tushar-sahani_TusharSahani_2125cs1072_2/GettingStarted/IsANumberPrime.java
package coderspear;

import java.util.*; 

public class IsANumberPrime {
    public static void main(String[] args) {
       Scanner scn = new Scanner(System.in);
       int t = scn.nextInt();
       while(t-->0){
           int n = scn.nextInt();
           int count =0;
           for(int i=2;i*i<=n;i++)
           {
               if(n%i==0)
               {
                count++;
               }
           }
           if(count>0){
            System.out.println("not prime");
           }
           else{
            System.out.println("prime");
           }
       }
       scn.close();
    }
}
